keps/sig-api-machinery/3156-http3/kep.yaml

```diff
@@ -0,0 +1,32 @@
+title: HTTP3
+kep-number: 3156
+authors:
+  - "@aojea"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-api-machinery
+  - sig-network
+status: deferred
```

Comment:
From the status options that come in the template,  deferred seems the best fit

Reply:
Yeah, this seems fine, is there a place we can put the reason? (Since future readers may not come back to this PR to see the comments)

Reply:
it seems this file is validated in ci, I've added a disclaimer in caps letter just at the beginning of the summary